_posts/2022-04-26-hhvm-4.159.markdown

```diff
@@ -0,0 +1,21 @@
+---
+title: "HHVM 4.159"
+layout: post
+author: atry
+category: blog
+---
+
+HHVM 4.159 is released! This release marks the end of support of 4.153; 4.154 -
```

Comment:
4.153 is an lts release. It is not losing support this week.

Reply:
Thanks for the reminder!